NCEP/gdas_utility.py

```diff
@@ -15,25 +16,27 @@
 from botocore.config import Config
 from botocore import UNSIGNED
 import argparse
-from pysolar.util import extraterrestrial_irrad
-from joblib import Parallel, delayed
-import pytz
-
+import requests
+from bs4 import BeautifulSoup
 
 
 class GFSDataProcessor:
-    def __init__(self, start_datetime, end_datetime, output_directory=None, download_directory=None, keep_downloaded_data=True):
+    def __init__(self, start_datetime, end_datetime, num_pressure_levels=13, download_source='nomads', output_directory=None, download_directory=None, keep_downloaded_data=True):
```

Comment:
I want to confirm, the default "download_source" is nomads?

Reply:
yes